mfa/oath/src/main/kotlin/com/pingidentity/mfa/oath/OathClient.kt

```diff
@@ -24,13 +21,13 @@ import kotlin.coroutines.coroutineContext
  * Implementation of OathClient that provides OATH functionality.
  * This client handles TOTP and HOTP credential management and code generation.
  *
- * @param configuration The MFA configuration.
+ * @param configuration The OATH configuration.
  * @param storage The OathStorage implementation to use. If null, a default SQLOathStorage will be created.
  */
 class OathClient(
```

Comment:
Suggest to provide one way to construct the object, now we have contructor, create() and invoke (DSL), suggest to keep the invoke only